README.md

```diff
@@ -1,4 +1,26 @@
-# Gender Pay Gap service

```

Comment:
This format was taken from [GDS's guide for writing READMEs](https://docs.publishing.service.gov.uk/manual/readmes.html).